nixos/tests/cassandra.nix

```diff
@@ -9,13 +9,16 @@ let
   jmxRolesFile = ./cassandra-jmx-roles;
   jmxAuthArgs = "-u ${(builtins.elemAt jmxRoles 0).username} -pw ${(builtins.elemAt jmxRoles 0).password}";
   jmxPort = 7200;  # Non-standard port so it doesn't accidentally work
+  jmxPortS = toString jmxPort;
```

Comment:
is this like jmx ports? the case like `PortS` seems incorrect.

Reply:
Yeah okay, I changed it to `jmxPortStr`. That should be clearer :)